### PR TITLE
Add aw

### PR DESCRIPTION
**Pull request by axelssonHakan**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,15 +2,18 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/node:10-stretch-browsers-legacy
+      - image: circleci/node:10
```
Comment (peol) [inline .circleci/config.yml]: 😍

File: .circleci/config.yml [hunk 2]
```diff
@@ -21,13 +24,28 @@ jobs:
...
+            DocID=$DocID npm run test:e2e
 
       - save_cache:
           paths:
             - node_modules
-          key: npm-dependency-cache-{{ checksum "package.json" }}
+          key: npm-dependency-cache-{{ checksum "package-lock.json" }}
+
+      - store_artifacts:
+          path: test/e2e/__artifacts__
```
Comment (peol) [inline .circleci/config.yml]: Why store these, aren't they supposed to be in the git repo? Or are these for CCI artefacts when something fails?

File: .eslintrc.json [hunk 3]
```diff
@@ -11,5 +11,11 @@
     "jsx-a11y/no-autofocus": 0,
     "import/no-named-as-default": 0,
     "max-len": 0
+  },
+  "globals": {
```
Comment (peol) [inline .eslintrc.json]: Add an extending `eslintrc.json` inside the `test/` folder, these are not valid globals in `src/`

File: test/comp/splash.spec.jsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Splash from '../../src/components/splash';
+
+describe('Splash', () => {
+  it('should renders docList correctly', () => {
```
Comment (peol) [inline test/comp/splash.spec.jsx]: `render`*

File: test/e2e/grid.spec.js [hunk 5]
```diff
@@ -0,0 +1,21 @@
+/* global page */
```
Comment (peol) [inline test/e2e/grid.spec.js]: Remove these, should be in eslintrc

File: test/e2e/grid.spec.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+const host = process.env.CI === 'true' ? 'localhost' : 'host.docker.internal';
+const engine = process.env.CI === 'true' ? 'localhost' : 'qix-engine';
+const app = process.env.DocID || 'drugcases.qvf';
+
+const url = `http://${host}:1234/?engine_url=ws://${engine}:9076/${app}`;
+const OPTS = {
+  artifactsPath: 'test/e2e/__artifacts__/',
+};
+
+describe('Catwalk', () => {
```
Comment (peol) [inline test/e2e/grid.spec.js]: `catwalk`*

File: test/unit/logic.spec.js [hunk 7]
```diff
@@ -0,0 +1,10 @@
+const logic = require('../../src/logic/logic');
+const { data } = require('./sample-data');
+
+describe('QueryModel', () => {
+  it('fields should return a valid object', () => {
```
Comment (peol) [inline test/unit/logic.spec.js]: `it('should return a valid object for a field',`?

File: .circleci/config.yml (hunk 2 above)
Comment (axelssonHakan) [reply .circleci/config.yml]: Exactly - they will be possible to view in the circleCI UI. So you don't need to ssh into the build container to view the images (baseline/diff/regression)

File: test/comp/aw.config.js [hunk 8]
```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  glob: 'test/comp/**/*.spec.jsx',
+  // coverage: true,
+  src: ['src/components/**/*.jsx'],
+  mocks: [
+    ['**/*.{svg,css,scss,pcss}'],
+    // ['**/src/enigma/config.js', () => {}],
```
Comment (wennmo) [inline test/comp/aw.config.js]: remove?

**Pull request merged**